baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: string, value: any, fileName?: string): void;
```

Comment:
@Kovensky Is there another area beyond just as a typedef where it might need to be added? I just updated my PR with the typedef for FormDataEntryValue but the test is failing with the following error `Error: The baseline file verifyDefaultLIb_dom.errors.txt has changed.` which is both a file that I don't seem to have and also one I suspect I shouldn't touch. I can't seem to figure out what in my changes would have prompted that but I'm guessing there is another area of the code where I would need to add FormDataEntryValue?